### PR TITLE
Added support for multi monitor configurations; the screen captured w…

### PR DESCRIPTION
**Pull request by gkpln3**
Added support for multi-monitor configurations; the screen captured will always be the one with the mouse in it.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: timelapse/recorder.py [hunk 1]
```diff
@@ -2,7 +2,16 @@
 import subprocess  # Taking screenshot
 import datetime  # Filename
 from multiprocessing import Process, Event
+from PyObjCTools import AppHelper
+from AppKit import *
```
Comment (cmangla) [inline timelapse/recorder.py]: Better to name and import only what you're using rather than importing '*'.

File: timelapse/recorder.py [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
 
+def getScreenIndexWithMouse():
```
Comment (cmangla) [inline timelapse/recorder.py]: I'm not sure about this project's naming conventions, but I believe PEP8 requires function names to lower case and `_` separated.

**Comment by cmangla** [comment]
If you run the project regression tests in your branch through github actions, the linter should pick up any simple issues like naming conventions.

**Comment by mre** [comment]
@gkpln3, thanks for the PR. That sounds very promising.
@cmangla thanks for the initial review. It looks like your concerns were addressed now. Do you have time to give it a spin on your local machine to see if it works as expected? If not, I could do it but it might take a bit longer.

**Comment by cmangla** [comment]
Yes @mre , they are indeed resolved now. Thanks @gkpln3 .

@mre I'm currently just working from my laptop so I don't have a second monitor to test this with. Will see if I can borrow one, but not sure.

**Comment by mre** [comment]
Ah no worries. I had a spare screen and set it up to test the PR. Works like a charm. Merging!
Thanks to both of you. 😊 👍

**Pull request merged**